### PR TITLE
fix: floating point issues

### PR DESCRIPTION
**Pull request by caminoezequiel**
# What?
Fixes fractional number for `PB_Vector3` in protobuf

# Why?
Setting a fractional number in the Transform can lead to unexpected results

### CODE REVIEW

File: kernel/packages/shared/proto/engineinterface_pb.d.ts [hunk 1]
```diff
@@ -1326,6 +1326,12 @@ export class PB_AvatarShape extends jspb.Message {
   getUsedummymodel(): boolean
   setUsedummymodel(value: boolean): void
 
+  getExpressiontriggerid(): string
```
Comment (nchamo) [inline kernel/packages/shared/proto/engineinterface_pb.d.ts:1329]: What are all these changes?

File: unity-client/Assets/Scripts/MainScripts/DCL/Models/Protocol/Engineinterface.cs [hunk 2]
```diff
@@ -3423,16 +3423,16 @@ public sealed partial class PB_Vector3 : pb::IMessage<PB_Vector3> {
           default:
             _unknownFields = pb::UnknownFieldSet.MergeFieldFrom(_unknownFields, input);
             break;
-          case 13: {
-            X = input.ReadFloat();
+          case 9: {
```
Comment (nchamo) [inline unity-client/Assets/Scripts/MainScripts/DCL/Models/Protocol/Engineinterface.cs:3426]: Where do these numbers come from? Is this code autogenerated?

**Review by nchamo — COMMENTED** [review]
Looks good, just a few questions 😄

File: unity-client/Assets/Scripts/MainScripts/DCL/Models/Protocol/Engineinterface.cs (hunk 2 above)
Comment (BrianAmadori) [reply unity-client/Assets/Scripts/MainScripts/DCL/Models/Protocol/Engineinterface.cs:3426]: Yep, this should be autogenerated by the protobuf compiler

**Comment by BrianAmadori** [comment]
Congratulations on your first PR 🙂  

Now, are we sure that this actually fixes the precision issue? How so? Because floating point precision errors are related with using numbers with big integer components, and in that test this seems to be not the case.

Also, the position deviation shown in the tests is very large, so this makes the case even more suspicious.

File: kernel/packages/shared/proto/engineinterface_pb.js [hunk 3]
```diff
@@ -4837,21 +4837,21 @@ proto.engineinterface.PB_Vector3.serializeBinaryToWriter = function(message, wri
   var f = undefined;
   f = message.getX();
   if (f !== 0.0) {
-    writer.writeFloat(
+    writer.writeDouble(
```
Comment (BrianAmadori) [inline kernel/packages/shared/proto/engineinterface_pb.js:4840]: If this change is really necessary, please make sure to modify the `.proto` templates too so the code is auto-generated as intended.

**Review by BrianAmadori — CHANGES_REQUESTED** [review]

File: kernel/packages/shared/proto/engineinterface_pb.d.ts (hunk 1 above)
Comment (caminoezequiel) [reply kernel/packages/shared/proto/engineinterface_pb.d.ts:1329]: @nchamo theses methods were autogenerated when I ran [protoc](https://github.com/decentraland/explorer/pull/1197/files/1acb6f1e98f6bb1eb9977d676520df1f56d5623c#diff-b3ea3aa7518ce3b22deb6fcd4967e0dd)

File: kernel/packages/shared/proto/engineinterface_pb.js (hunk 3 above)
Comment (caminoezequiel) [reply kernel/packages/shared/proto/engineinterface_pb.js:4840]: @BrianAmadori yes, I did. I updated this file [.proto](https://github.com/decentraland/explorer/pull/1197/files/1acb6f1e98f6bb1eb9977d676520df1f56d5623c#diff-ef819b8b253c3a3525fe2fcf12e43ce2) I don't know if there is another file where I should change it, do you know? 

After changed `.proto` file I ran these [commands](https://github.com/decentraland/explorer/pull/1197/files/1acb6f1e98f6bb1eb9977d676520df1f56d5623c#diff-b3ea3aa7518ce3b22deb6fcd4967e0dd) please let me know if it is ok.

**Comment by caminoezequiel** [comment]
replaced by: https://github.com/decentraland/explorer/pull/1199

**Pull request closed without merging**